src/coldcard.test.js

```diff
@@ -0,0 +1,148 @@
+import {ColdcardMultisigWalletConfig, WALLET_CONFIG_VERSION} from './coldcard';
+
+describe("ColdcardMultisigWalletConfig", () => {
+  const jsonConfigUUID = {
```

Comment:
Two other suggestions:
1. I'm not sure the quotations around the key names are necessary since this is still technically an object and not a JSON string, so it could just make this easier to maintain.
2. It might be a good idea to put this in a `beforeEach` block so that the value is reset after each test to avoid tests polluting each other.

Reply:
As for 1 - noted - I just pasted in what comes from unchained external spend JSON. updated
2 - definitely.